src/plugins/inspectorDataVisualization/DataVisualization.vue

```diff
@@ -0,0 +1,186 @@
+<!--
+ Open MCT, Copyright (c) 2014-2023, United States Government
+ as represented by the Administrator of the National Aeronautics and Space
+ Administration. All rights reserved.
+
+ Open MCT is licensed under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations
+ under the License.
+
+ Open MCT includes source code licensed under additional open source
+ licenses. See the Open Source Licenses file (LICENSES.md) included with
+ this source code distribution or the Licensing information page available
+ at runtime from the About dialog for additional information.
+-->
+
+<template>
+  <div
+    class="c-data-visualization-inspect-properties c-inspector__data-pivot c-data-visualization-inspector__flex-column"
+  >
+    <div class="c-inspect-properties">
+      <div class="c-inspect-properties__header">Data Visualization</div>
+    </div>
+
+    <div v-if="isLoading" class="c-inspector__data-pivot-placeholder">Loading...</div>
+
+    <div v-else-if="hasDataRanges">
+      <div
+        v-if="selectedDataRange !== undefined && hasDescription"
+        class="c-inspector__data-pivot-coordinates-wrapper"
+      >
+        <span
+          class="c-tree__item__type-icon c-object-label__type-icon"
+          :class="description.icon"
+        ></span>
+        <span class="c-inspector__data-pivot-coordinates">
+          {{ description.text }}
+        </span>
+      </div>
+
+      <select class="c-inspector__data-pivot-range-selector" v-model="selectedDataRangeIndex">
+        <option
+          v-for="(dataRange, index) in descendingDataRanges"
+          :key="index"
```

Comment:
key should be something else than index, preferably an id or something with the dataRange object

Reply:
Thanks for that feedback @evenstensberg! This was already merged to master with the existing code, and this PR is cherry-picking into a release that we're cutting this week. I'll leave the code as-is for now, but we'll definitely work to get this cleaned up in master. Thank you for your help!